static/app/views/starfish/views/webServiceView/starfishLanding.tsx

```diff
@@ -39,7 +51,17 @@ export function StarfishLanding(props: Props) {
       <PageErrorProvider>
         <Layout.Header>
           <Layout.HeaderContent>
-            <Layout.Title>{t('Web Service')}</Layout.Title>
+            <Layout.Title>
+              {selectedProject ? (
+                <ProjectBadge
```

Comment:
Can we add it to DB and API modules and generic span view also? those should be the only two other places since the sub pages should be scoped down to a txn/span at that poing

Reply:
Will do 👍